proxy/src/http/org.rs

```diff
@@ -0,0 +1,256 @@
+//! Endpoints for Org.
+
+use serde::ser::SerializeStruct as _;
+use serde::{Deserialize, Serialize, Serializer};
+use std::sync::Arc;
+use tokio::sync::RwLock;
+use warp::document::{self, ToDocumentedType};
+use warp::{path, Filter, Rejection, Reply};
+
+use crate::avatar;
+use crate::notification;
+use crate::registry;
+
+/// The org with identifier and avatar
+pub struct Org {
+    /// The unique identifier of the org
+    id: String,
+    /// Generated fallback avatar 
+    avatar_fallback: avatar::Avatar
+}
+
+/// Combination of all org routes.
+pub fn filters(
+    registry: Arc<RwLock<registry::Registry>>,
+    subscriptions: notification::Subscriptions,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    get_filter().or(register_filter(registry, subscriptions))
+}
+
+/// POST /orgs/register
```

Comment:
The only real write operation we have on top-level orgs is the registration, so synonymous with creation. Would say we can just do `POST /orgs` for that endpoint.